server/game/gamesteps/uiprompt.js

```diff
@@ -15,7 +15,6 @@ class UiPrompt extends BaseStep {
 
     complete() {
         this.completed = true;
-        this.game.resetClocks();
```

Comment:
We don't need to reset the clocks when a prompt is complete.  This happens when the player is not the active player anyway.

The flow used to be:

- Action is taken
- Stop all clocks
- Start active player clock
- Stop inactive player clock

Flow is now:
- Action is taken
- Start active player clock (does nothing if it's already started)
- Stop inactive player clock (does nothing if it's already stopped)